src/lib.rs

```diff
@@ -1603,15 +1606,13 @@ impl RtcConfig {
     }
 
     /// The configured DtlsCert.
-    ///
-    /// The certificate is uniquely created per new RtcConfig.
-    pub fn dtls_cert(&self) -> &DtlsCert {
-        &self.dtls_cert
+    pub fn dtls_cert(&self) -> Option<&DtlsCert> {
+        self.dtls_cert.as_ref()
```

Comment:
I think it be worth to say that this is a _previously set_ dtls cert and that if you need the cert in use, go to direct api (and link to the call).